lib/pages/chat/events/download_video_state.dart

```diff
@@ -0,0 +1,6 @@
+enum DownloadState {
```

Comment:
```suggestion
enum DownloadVideoState {
```
what diferrent between waiting and loading
can change waiting to initial ?